crmprtd/networks/ec/normalize.py

```diff
@@ -1,7 +1,41 @@
+import logging
+import yaml
+
+from importlib.resources import files
+from crmprtd import Row
 from crmprtd.swob_ml import normalize as swob_ml_normalize
 
+log = logging.getLogger(__name__)
+
 
 def normalize(file_stream):
-    return swob_ml_normalize(
+
```

Comment:
I'm always skeptical when a patch adds LOCs to our repo. :) The approach you've taken here is reasonable, but I don't think that this is (or will be) specific to ECCC. Could you take what you've done here and incorporate it into the `swob_ml` module? Then if other networks change their variable names in the future, we will have an easy place to add variable mappings.